### ISSUE TITLE
Ctrl-^ not work

### ISSUE THREAD

**Issue by meijieru** [onivim/oni#1194]
{num}<Ctrl-^> should switch to the {num} buffer, but it does not work in oni

**Comment by badosu** [comment]
I can confirm that lately I have not been able to use `<C-^>` as well, using the lates stable version

**Comment by bryphe** [comment]
Thanks for the report, @meijieru and @badosu! This seems to be related to the keyboard layout - I'm wondering if it is being picked up as a dead key.

Would it be possible to grab the logs from the console when you execute that keystroke?
- `Control+Shift+P` -> `Open DevTools`
- Enter `Oni.log.enableDebugLogging()`
- Switch back to Oni and execute the key press. You should see some [key event] logs in the console, like this:
![image](https://user-images.githubusercontent.com/13532591/34543472-d30e73ea-f095-11e7-9fb6-102965920aad.png)

This can help narrow down how the key press is being resolved / what's going wrong.

In addition, knowing what the `keyboard-layout` module sees as the current language, would be helpful, too:
![image](https://user-images.githubusercontent.com/13532591/34543515-11439c8a-f096-11e7-9701-ba180c2047f0.png)
You can check that by running the `require("keyboard-layout").getCurrentKeyboardLanguage()`

Thanks the report & your help!

**Comment by meijieru** [comment]
@bryphe Sorry, I am not familiar with node js, so where should I enter this command `Oni.log.enableDebugLogging()`? I try it within `console`, but it says this is undefined.

**Comment by meijieru** [comment]
Anything new?

**Comment by CrossR** [comment]
What version of Oni are you currently using?

I'm able to use  `<Ctrl+^>` on the latest build, so its possible its working now.

If that doesn't work, the place you'd enter `Oni.log.enableDebugLogging()` and `require("keyboard-layout").getCurrentKeyboardLanguage()` is in the Chrome dev tools. You access that with `<Ctrl+Shift+P>` and typing Dev tools.

**Comment by badosu** [comment]
@CrossR I am using the latest stable and this issue persists. This is the output on a normal Oni installation:

```
< require("keyboard-layout").getCurrentKeyboardLanguage()
> "us [0]"
```

**Comment by CrossR** [comment]
Hm, mine works on US and UK keyboards, but I've only tested on Windows.

What is the output of the verbose log?

I get the following for `3<Ctrl+^>`.
![image](https://user-images.githubusercontent.com/10038688/37624323-a947fa6c-2bbf-11e8-8686-9a106cf255f4.png)

**Comment by badosu** [comment]
Ok, it seems that `<Ctrl-^>` is working, but the help text makes it seem like `<Ctrl-6>` should work as well (that's how I used it previously).

I am fine using `<Ctrl-^>` if that's not a bug.

```
							*CTRL-^* *CTRL-6*
CTRL-^			Edit the alternate file.  Mostly the alternate file is
			the previously edited file.  This is a quick way to
			toggle between two files.  It is equivalent to ":e #",
			except that it also works when there is no file name.

			If the 'autowrite' or 'autowriteall' option is on and
			the buffer was changed, write it.
			Mostly the ^ character is positioned on the 6 key,
			pressing CTRL and 6 then gets you what we call CTRL-^.
			But on some non-US keyboards CTRL-^ is produced in
			another way.
```

This is my `<Ctrl-6>` output:

![screenshot from 2018-03-19 19-02-42](https://user-images.githubusercontent.com/347552/37624781-2572668a-2ba8-11e8-90e0-7fdf858c3e1b.png)

**Comment by CrossR** [comment]
I'm a bit of a buffers newbie, but the behaviour for me (again Windows only currently, I should probably test on my Arch box) is the same behaviour as in nvim-qt and nvim in the terminal.

For other platforms though, we could have a keyboard capture issue, but it looks as if `<Ctrl-6>` is being picked up correctly.

**Comment by badosu** [comment]
@CrossR Makes sense, I've been using Oni for a while so maybe my memory on my previous workflow is not as good (which works with `<C-^>` now).

The pending issue now if for @meijieru to check if it's working for him and to provide his debug details.

**Comment by meijieru** [comment]
@badosu <C-^> works. I am custommed to use <C-6>.
So is it possible to enable <c-6> as well?

**Comment by badosu** [comment]
@meijieru It is working for me at the moment without any configuration, can you confirm if you still can't use it in Oni on the last stable release (0.3.1)?

**Comment by meijieru** [comment]
@badosu Github conceal the <Ctrl-6>. I am using 0.3.1 now, and <Ctrl-^> works for me, while <Ctrl-6> not.

**Comment by CrossR** [comment]
Looks like this might have been "sorted" over in #2002.
We could use this ticket to track adding a default keybind, if we think its needed.

Though we could just end up adding a keybind that causes people issues, so we'd have to be careful to make sure its easily disabled....May just make more sense to add documentation around it instead?
